pyproject.toml

```diff
@@ -46,6 +46,7 @@ crispy-forms-gds = "^0.3.1"
 django-gov-notify = { git = "https://github.com/i-dot-ai/django-gov-notify", branch = "support-python-3-12" }
 sentry-sdk = "^2.0.1"
 torch = "~2.2"
+jsonschema = {extras = ["format"], version = "^4.22.0"}
```

Comment:
Is this actually used in the code anywhere? Useful for testing, maybe should stay (or go into the dev dependencies?).

Reply:
Yes — it's necessary to get good error messages out of the validator in `ConsultationUploadForm`

Reply:
(confusingly, it's not used explicitly, it just magically enhances the output of `jsonschema.validate()`